app/views/scans/_form_additional.html.erb

```diff
@@ -1,5 +1,11 @@
-<%= f.text_field :page_range, help: 'Example: 1-5, 8, 7-13' %>
+<div class="form-group">
+  <%= f.label :page_range, class: label_column_class + ' control-label ' %>
```

Comment:
I did try this:

`class: '#{label_column_class} control-label’`

But it was rendered literally:

`<label class="#{label_column_class} control-label" for="request_page_range">Page range</label>`


On Aug 19, 2015, at 3:13 PM, Jessie Keck <notifications@github.com<mailto:notifications@github.com>> wrote:


In app/views/scans/_form_additional.html.erb<https://github.com/sul-dlss/sul-requests/pull/310#discussion_r37475444>:

> @@ -1,5 +1,11 @@
> -<%= f.text_field :page_range, help: 'Example: 1-5, 8, 7-13' %>
> +<div class="form-group">
> +  <%= f.label :page_range, class: label_column_class + ' control-label ' %>


Not to be pedantic, but could you use string interpolation here instead of concatenation? Check out this example: https://repl.it/BCec

If you try to concatenate a nil variable and a string you'll get an error. However; using interpolation to_s will be called on the nil variable and an empty string will be returned. nil.to_s => ""

—
Reply to this email directly or view it on GitHub<https://github.com/sul-dlss/sul-requests/pull/310/files#r37475444>.